File: Selection Sort/selection_sort.py
def selectionSort(arr):
    for i in range(len(arr)):
        minima = arr[i]
        min_ind = i
        for j in range (i+1, len(arr)):
            if arr[j] < minima:
                minima = arr[j]
                min_ind = j
        arr[i], arr[min_ind] = arr[min_ind], arr[i]
string = input().split(" ")
data = [int(item) for item in string]
selectionSort(data)
print(data)

File: Selection Sort/SelectionSort.java
public class SelectionSort {
  public static void sort(int[] input) {

        int inputLength = input.length;

        for (int i = 0; i < inputLength - 1; i++) {

            int min = i;
            for (int j = i + 1; j < inputLength; j++) {
                if (input[j] < input[min]) {
                    min = j;
                }
            }
            int temp = input[i];
            input[i] = input[min];
            input[min] = temp;
        }

    }

}
